pandas/core/algorithms.py

```diff
@@ -981,14 +981,60 @@ def select_n_frame(frame, columns, n, method, keep):
     -------
     nordered : DataFrame
     """
-    from pandas.core.series import Series
+    from pandas import Int64Index
     if not is_list_like(columns):
         columns = [columns]
     columns = list(columns)
```

Comment:
though keep the message for Series pretty much the same (actually it IS tested).